### DISCUSSION TITLE
<input type="time" list="data-list"> does not seem to work

### DISCUSSION THREAD

**Discussion by sureshvv** [bigskysoftware/htmx#1329] [Q&A]
I have an input type="date" control and an input type="time" control.

I am using the list attribute of the time control to show a list of valid times.

It is dependent on the date.

Does not work even after the list is updated.

How can I refresh the input?

**Comment by sureshvv** [comment]
It works mostly. The time format is locale specific

**Reply by andryyy** [reply]
Can you post an example?
